src/libpriv/rpmostree-types.h

```diff
@@ -24,6 +24,7 @@
 typedef enum {
   RPM_OSTREE_PKG_TYPE_BASE,
   RPM_OSTREE_PKG_TYPE_LAYER,
+  RPM_OSTREE_PKG_TYPE_UNKNOWN,
```

Comment:
Hmm.  Why don't we just entirely kill the enum if we're not using it anymore?  Is `rpm_diff_add_layered_diff` doing anything anymore?

Reply:
The issue is that it's part of the D-Bus API now, so we can't change the signature of the property. :( E.g. see: https://gitlab.gnome.org/GNOME/gnome-software/blob/fca34d6e8e001ab94e393feb7bde0e857447c98c/plugins/rpm-ostree/gs-plugin-rpm-ostree.c#L327. We could also just mark it all as `BASE` instead of a new `UNKNOWN`?

I still think it can be a useful marker though. E.g. I can imagine g-s using it to make it easy to see what base pkgs changed vs. layered. For example, a layered pkg that now pulls in more deps would immediately jump out. Or enhancing `rpm-ostree status` advisories/diff printing.